### ISSUE TITLE
Image Occlusion: Merge of two hiding boxes does not work

### ISSUE THREAD

**Issue by NoFunCode** [remnoteio/remnote-issues#1336]
**Describe the bug**
Somehow remnote does not save the image occlusion hiding boxes I wanted to merge

**To Reproduce**
1. Insert an image
2. Create two independet boxes for image occlusion "A" and "B"
3. Do this card in the queue
4. Edit the image occlusion and merge "A" and "B" to one Box either both "A" or both "B"
5. They always stay "A" and "B" and were not merged

**Expected behavior**
It should be two boxes with either "A" "A" or "B" "B"

**Desktop (please complete the following information):**
 - OS: Windows
 - Browser remnote application
 - Version 1.4.*

**Smartphone (please complete the following information):**
 - Device: iPhone 11 Pro
 - OS: iOS 15
 - Browser safari
 - Version 15

**Comment by NoFunCode** [comment]
UPDATE: I cannot even create a single image occlusion box. Pretty sad.

**Comment by imageobject** [comment]
> UPDATE: I cannot even create a single image occlusion box. Pretty sad.

i'm having the same issue!! it's inconsistent--sometimes it works but most of the time they disappear after i click "save"

**Comment by NoFunCode** [comment]
UPDATE: Newly added images work. But the old images have the error described above.

**Comment by imageobject** [comment]
hot tip!! thank you

the new images seem to be working, BUT ONLY when i create the occlusion by clicking the three dots in the upper right hand corner. if i command-click and generate occlusions that way, they still disappear when i save. 
<img width="705" alt="Screen Shot 2021-09-24 at 6 58 43 PM" src="https://user-images.githubusercontent.com/91297124/134747704-f4cdc598-48c1-4d6f-9bfe-f3c3ec8c7b74.png">

**Comment by lovely-sad** [comment]
> hot tip!! thank you
> 
> the new images seem to be working, BUT ONLY when i create the occlusion by clicking the three dots in the upper right hand corner. if i command-click and generate occlusions that way, they still disappear when i save.
> <img alt="Screen Shot 2021-09-24 at 6 58 43 PM" width="705" src="https://user-images.githubusercontent.com/91297124/134747704-f4cdc598-48c1-4d6f-9bfe-f3c3ec8c7b74.png">

I found the solution! If i click image it self then click right top 3 dot then delete it normally. It works because it focused the image. When you dont click image if you try to delete it, it scrolls to another image. Because it is not focused on it

(my english is not good sorry)

**Comment by NoFunCode** [comment]
@lovely-sad you are absolutely right. In order to make it work: first click on the image so that it is focused and than use the three dots to open the image occlusion part. **Not** with the shortcut `CTRL + Click`. Then it works. Thanks mate!!!
